src/pytest_plugins/filler/filler.py

```diff
@@ -335,6 +344,9 @@ def pytest_configure(config):
         called before the pytest-html plugin's pytest_configure to ensure that
         it uses the modified `htmlpath` option.
     """
+    if not hasattr(config, "fill_mode"):
+        config.fill_mode = FillMode.CONSENSUS
```

Comment:
This should be as an `else` branch in `pytest_configure` in `src/pytest_plugins/filler/benchmarking.py`, because otherwise we require to import this `filler.py` during execute.